pagopa/io-app
# Fix E2E setup

## Short description
This PR forces the `it_IT` language and locale to the current running application, to avoid language misconfiguration during tests. 

## List of changes proposed in this pull request
See commit list

## How to test
E2E test should run fine ([GA action success run here](https://github.com/pagopa/io-app/actions/runs/5523172958)).
Try run E2E tests on your machine. All tests must succeed.
